swaggen/src/main/java/utils/io/FileMapper.java

```diff
@@ -37,15 +47,79 @@
 	 * Maps a Java object to a YAML file.
 	 * 
 	 * @param filename the name of the YAML file
-	 * @param object the Java object
+	 * @param newSwagger the Java object, typically swagger object 
+	 * @param <T> The type
+	 * 
+	 * @throws JsonGenerationException JsonGenerationException
+	 * @throws JsonMappingException JsonMappingException
+	 * @throws IOException IOException
+	 */
+	public static <T> void classToYaml(String filename, T newSwagger)
+			throws JsonGenerationException, JsonMappingException, IOException {
+		ObjectMapper objectMapper = new ObjectMapper(new YAMLFactory());
+		File file = new File(filename);
+
+		checkExistingEndpoints(newSwagger, file, objectMapper);
+
+		if (newSwagger instanceof Swagger) {

```

Comment:
Make sure `instanceof` is completely necessary. I suspect that it is not. See comment below.